city_scrapers/spiders/chi_ohare_noise.py

```diff
@@ -0,0 +1,212 @@
+import re
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import (
+    CANCELLED,
+    COMMISSION,
+    COMMITTEE,
+    NOT_CLASSIFIED,
+    PASSED,
+    TENTATIVE,
+)
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from scrapy import Request
+
+
+class ChiOhareNoiseSpider(CityScrapersSpider):
+    name = "chi_ohare_noise"
+    agency = "Chicago O'Hare Noise Compatibility Commission"
+    timezone = "America/Chicago"
+
+    class ChiOhareNoiseSubSpider1:
+        def parse(self, response):
+            """
+            `parse` should always `yield` Meeting items.
+
+            Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+            needs.
+            """
+            for item in response.css(".ev_td_li"):
+                surl = self._parse_url(item)
+                yield Request(url=response.urljoin(surl), callback=self._parse_details)
+
+            next_page = response.xpath("//div[@class='previousmonth']/a/@href").get()
+            if next_page is not None:
+                yield response.follow(response.urljoin(next_page), callback=self.parse)
+
+        def _parse_details(self, response):
+            stime = self._parse_start(response)
+            meeting = Meeting(
+                title=self._parse_title(response).replace("CANCELLED ", "").strip("- "),
+                description=self._parse_description(response),
+                start=stime,
+                end=stime + timedelta(hours=1),
```

Comment:
We can leave this as `None` if we aren't parsing it and potentially pass a string to the `time_notes` kwarg with any details about scheduling